modules/apps/account/account-service/src/main/java/com/liferay/account/internal/instance/lifecycle/AddDefaultAccountRolesPortalInstanceLifecycleListener.java

```diff
@@ -129,20 +140,76 @@ private void _addRole(long userId, String roleName) throws PortalException {
 			HashMapBuilder.put(
 				LocaleThreadLocal.getDefaultLocale(), roleName
 			).build(),
-			null, RoleConstants.TYPE_REGULAR, null, null);
+			_getDescriptionMap(roleName), RoleConstants.TYPE_REGULAR, null,
+			null);
 	}
 
 	private boolean _exists(String roleName) {
 		Role role = _roleLocalService.fetchRole(
 			CompanyThreadLocal.getCompanyId(), roleName);
 
 		if (role != null) {
+			Map<Locale, String> descriptionMap = _getDescriptionMap(roleName);
+
+			if (!Objects.equals(role.getDescriptionMap(), descriptionMap)) {
+				role.setDescriptionMap(descriptionMap);
+
+				_roleLocalService.updateRole(role);
+			}
+
```

Comment:
You're right. The edited descriptions won't persist when the module refreshes. The reason I did this is because we're localizing the descriptions with language keys, and this ensures that we get the translated keys into the descriptions when they get updated. Another option is to only provide the English description like other system roles.